Classes/music_album.rb

```diff
@@ -1,4 +1,14 @@
 require_relative 'Classes/item'
 
 class MusicAlbum < Item
+  def initialize(on_spotify)
+    super()
```

Comment:
![music album](https://user-images.githubusercontent.com/86690432/234052155-2e1ab1ac-e795-420e-8c71-607da9e9291c.png)

- From your code, I can see that you are calling the Super method on the `MusicAlbum` class without giving it any parameters. This would break your code because the Parent class requires argument. So I woul suggest you add the necessary parameters to the Super() method while calling it in the child class.

- Here is the Parent class and as you can see, the `initialize method requires some parameters`

![item album](https://user-images.githubusercontent.com/86690432/234053057-18cfa85a-aaeb-495a-b853-c5775de1c00c.png)